src/flow.ts

```diff
@@ -68,7 +68,13 @@ async function redirectFlow(options): Promise<FlowResult>  {
     return Promise.reject('ERR_REDIRECT_FLOW_BAD_ARGS');
   }
 
-  const xmlString = inflateString(decodeURIComponent(content));
+  let xmlString: string
+
+  try {
+    xmlString = inflateString(decodeURIComponent(content));
+  } catch (e) {
+    throw new WrappedError('ERR_FAILED_INFLATION', e)
```

Comment:
See my other comment, I think
```suggestion
  } catch (cause) {
    throw new Error('ERR_FAILED_INFLATION', { cause })
```
should suffice